website/docs/r/iam_trusted_profiles_link.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profile_link"
+description: |-
+  Manages iam_trusted_profile_link.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profile_link
+
+Create, update, and delete an IAM trusted profile link. For more information, about IAM trusted profile, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link 
```

Comment:
Create, update, and delete an IAM trusted profile link. For more information, about IAM trusted profile, see https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link 

as

Create, update, or delete an IAM trusted profile link resource. For more information, about IAM trusted profile link, see [Create a link to a trusted profile](https://cloud.ibm.com/apidocs/iam-identity-token-api#create-link )